apps/accounts/serializers.py

```diff
@@ -69,3 +70,12 @@ def update(self, instance, validated_data):
             profile.linkedin_url = linkedin_url
             profile.save()
         return instance
+
+
+class CustomPasswordResetSerializer(PasswordResetSerializer):
+    def get_email_options(self):
+        super().get_email_options()
+        return {
```

Comment:
Add newline above this

Reply:
Hi @Ram81 bhaiya, do you mean after adding docstring to add a line, or above the class, I mean above this class I have already left 2 lines according to PEP8 guide.
Thanks

Reply:
super().get_email_options() I have removed this line, because this function is totally empty and is of no use. I have done all the tests.
Thanks